test/Chainweb/Test/Pact/ChainData.hs

```diff
@@ -228,12 +228,10 @@ data TestException = TestException
 instance Exception TestException
 
 testMemPoolAccess :: T.Text -> IO (Time Integer) -> MemPoolAccess
-testMemPoolAccess t iotime = MemPoolAccess
+testMemPoolAccess t iotime = mempty
     { mpaGetBlock = \validate bh hash _parentHeader -> do
         time <- f bh <$> iotime
         getTestBlock t time validate bh hash
-    , mpaSetLastHeader = \_ -> return ()
-    , mpaProcessFork = \_ -> return ()
     }
```

Comment:
I am not a fan of this kind of changes, since it makes reading the code more difficult, since there isn't is an obvious (at least to me) canonical Monoid instance for MempoolAccess.

Reply:
This is test code -- it's mainly to avoid future updates to these regions if we add another field to `MemPoolAccess`, I've already had to go through them all several times and I'm tired of it. I can change this to `emptyMempoolAccess` with `emptyMempoolAccess = mempty` if you'd prefer, and I actually in principle agree with you. I'd make the same argument about `Default`

Reply:
I agree, for test code it doesn't matter that much and coding convenience is of higher value than in production code. Let's leaf it as is.